MapboxNavigation/RouteMapViewController.swift

```diff
@@ -82,15 +82,15 @@ class RouteMapViewController: UIViewController {
         } else {
             setDefaultCamera(animated: false)
         }
-        
-        mapView.setUserLocationVerticalAlignment(.bottom, animated: false)
-        mapView.setUserTrackingMode(.followWithCourse, animated: false)
 
         UIDevice.current.addObserver(self, forKeyPath: "batteryState", options: .initial, context: nil)
     }
 
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
+        
+        mapView.setUserTrackingMode(.followWithCourse, animated: false)
+        mapView.setUserLocationVerticalAlignment(.bottom, animated: false)
```

Comment:
Is this a map SDK limitation? If so, can you whip up a reduced test case and file an issue in gl-native? Thanks!